### PR TITLE
Rubber gloves for janitors

### PR DESCRIPTION
**Pull request by UbaserB**
## About the PR

Once upon a time, a mere ideasguy mentioned the idea. Little did they know, it became a reality. Why do janitors have to clean with their bare hands? It's messy! Time to give them gloves...

_It looks nearly exactly like yellow (insul) gloves but longer._

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

image

**Changelog**

:cl: Ubaser
- add: Janitors now get rubber gloves for cleaning.

### CODE REVIEW

**Comment by KittenColony** [comment]
Merge this immediately

File: Resources/Textures/Clothing/Hands/Gloves/janitor.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"version": 1, "license": "CC-BY-SA-3.0", "copyright": "Drawn by Ubaser", "size": {"x": 32, "y": 32}, "states": [{"name": "icon"}, {"name": "equipped-HAND", "directions": 4}, {"name": "inhand-left", "directions": 4}, {"name": "inhand-right", "directions": 4}]}
```
Comment (deltanedas) [inline Resources/Textures/Clothing/Hands/Gloves/janitor.rsi/meta.json:1]: formatting

Comment (UbaserB) [reply Resources/Textures/Clothing/Hands/Gloves/janitor.rsi/meta.json:1]: i copied other glove s

**Comment by UbaserB** [comment]
i formatted it like you asked

**Pull request merged**